### PR TITLE
Complete unsuccessful signup functionality

### PR DESCRIPTION
**Pull request by NickPapayiannakis**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,17 +20,11 @@ gem 'uglifier', '>= 1.3.0'
...
-# gem 'redis', '~> 3.0'
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
 
+gem 'rails-controller-testing'
 # ReactJS gem
 gem 'react-rails', '~> 1.0'
 
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
```
Comment (bpassos-zz) [inline Gemfile:32]: Not sure Capistrano is what we're going with for deploying. Do you have strong reasons for this? Maybe webpack can offer a better all in on build and deploy. Ok to leave it for now.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <% provide(:title, @user.first_name) %>
-<div class="row">
-  <aside class="col-md-4">
```
Comment (bpassos-zz) [inline app/views/users/show.html.erb:3]: Are we using bootstrap? This class name rings the bell, or are you hoping that our grid system caters for these classes?

**Review by bpassos-zz — APPROVED** [review]
👏🏼

File: app/views/users/show.html.erb (hunk 2 above)
Comment (NickPapayiannakis) [reply app/views/users/show.html.erb:3]: Disregard, deleting (I keep letting bootstrap stuff slip in)

File: Gemfile (hunk 1 above)
Comment (NickPapayiannakis) [reply Gemfile:32]: Not sure how this made it through, will delete

**Pull request merged**